user_form_app/lib/app/feature/user_form/widgets/education_widget.dart

```diff
@@ -15,28 +17,40 @@ class EducationWdiget extends StatelessWidget {
                 Theme.of(context).textTheme.labelLarge!.copyWith(fontSize: 16)),
         const SizedBox(height: 15),
         DropdownButtonFormField(
-          items: [],
-          onChanged: (va) {},
+          items: educationList
```

Comment:
can we get the required list in the params instead of mapping here?
also if we can look at the localisation in the entire project.